.fides/fides.toml

```diff
@@ -22,7 +22,7 @@ analytics_opt_out = false
 
 [redis]
 host = "redis"
-password = "testpassword"
+password = "redispassword"
```

Comment:
Changed this to be extra careful - we were setting a default password in the Pydantic config (which is surprising!), so I wanted to be doubly sure that we weren't relying on that anywhere in our code.